nearswap/Cargo.toml

```diff
@@ -23,6 +23,7 @@ funty = "=1.1.0"
 #New version of simulation tests:
 # [dev-dependencies]
 near-sdk-sim = { git = "https://github.com/near/near-sdk-rs.git", tag="3.0.0" }
```

Comment:
```suggestion
near-sdk-sim = "=3.0.0-pre.2" // { git = "https://github.com/near/near-sdk-rs.git", tag="3.0.0" }
```

Reply:
![Screenshot from 2021-03-04 22-16-56](https://user-images.githubusercontent.com/26249903/110003455-68852c80-7d3c-11eb-92a1-21fd709bf944.png)
this is not working